code/game/machinery/doors/airlock.dm

```diff
@@ -39,9 +39,9 @@
 	var/secured_wires = 0
 	var/datum/wires/airlock/wires = null
 
-	var/open_sound_powered = 'sound/machines/door/covert1o.ogg'
+	var/open_sound_powered = 'sound/machines/airlock.ogg'
 	var/open_sound_unpowered = 'sound/machines/airlockforced.ogg'
-	var/close_sound_powered = 'sound/machines/door/covert1c.ogg'
+	var/close_sound_powered = 'sound/machines/airlockclose.ogg'

```

Comment:
I would like this to be reverted back.

Reply:
I'd suggest using hall1c and hall1o for close and open sounds respectively.

They fit well for the doors.